app.py

```diff
@@ -242,6 +243,25 @@ def __add_write_node_commands(self, context=None):
                     context=context,
                 )
             )
-            
-            
 
+        # Show the convert actions in the Menu if configured to do so
+        if self.get_setting('show_convert_actions'):
+
+            self.engine.register_command(
+                "Convert from Shotgun Write Nodes",
+                self.convert_to_write_nodes,
+                {
+                    "type": "context_menu",
+                    "icon": os.path.join(self.disk_location, "icon_256.png"),
+                    "context": context,
```

Comment:
what is the context key for? It is not documented in our API reference: https://developer.shotgunsoftware.com/tk-core/platform.html?highlight=register_command#sgtk.platform.Engine.register_command

I think this should be deleted?